src/ctia/graphql/schemas.clj

```diff
@@ -42,137 +44,167 @@
+                      :args    {:type  {:type (g/non-null Scalars/GraphQLString)}
+                                :value {:type (g/non-null Scalars/GraphQLString)}}
+                      :resolve (fn [_ args _ _] args)}
+   :sighting         {:type    SightingType
+                      :args    search-by-id-args
+                      :resolve (res/entity-by-id-resolver :sighting)}
+   :sightings        {:type    SightingConnectionType
+                      :args    (merge common/lucene-query-arguments
+                                      sighting/sighting-order-arg
+                                      p/connection-arguments)
+                      :resolve (res/search-entity-resolver :sighting)}
+   :target_record    {:type    TargetRecordType
+                      :args    search-by-id-args
+                      :resolve (res/entity-by-id-resolver :target-record)}
+   :target_records   {:type    TargetRecordConnectionType
+                      :args    (merge common/lucene-query-arguments
+                                      target-record/target-record-order-arg
+                                      p/connection-arguments)
+                      :resolve (res/search-entity-resolver :target-record)}
+   :tool             {:type    ToolType
+                      :args    search-by-id-args
+                      :resolve (res/entity-by-id-resolver :tool)}
+   :tools            {:type    ToolConnectionType
+                      :args    (merge common/lucene-query-arguments
+                                      tool/tool-order-arg
+                                      p/connection-arguments)
+                      :resolve (res/search-entity-resolver :tool)}
+   :vulnerability    {:type    VulnerabilityType
+                      :args    search-by-id-args
+                      :resolve (res/entity-by-id-resolver :vulnerability)}
+   :vulnerabilities  {:type    VulnerabilityConnectionType
+                      :args    (merge common/lucene-query-arguments
+                                      vulnerability/vulnerability-order-arg
+                                      p/connection-arguments)
+                      :resolve (res/search-entity-resolver :vulnerability)}
+   :weakness         {:type    WeaknessType
+                      :args    search-by-id-args
+                      :resolve (res/entity-by-id-resolver :weakness)}
+   :weaknesses       {:type    WeaknessConnectionType
+                      :args    (merge common/lucene-query-arguments
+                                      weakness/weakness-order-arg
+                                      p/connection-arguments)
+                      :resolve (res/search-entity-resolver :weakness)}})
+
+(defn- dash->underscore [k]
+  (-> k
+      name
+      (string/replace "-" "_")
+      keyword))
+
+(defn- disabled-entities
+  [{{:keys [enabled?]} :FeaturesService}]
+  (->> (entities/all-entities)
+       keys
+       (filter #(not (enabled? %)))))
```

Comment:
I would transfer this function to the FeatureService.

Reply:
I thought about that, but then decided not to do it at this point (requires changing schemas, etc.), maybe some other time if we see something like that needed. I'll keep in mind that we have it here and move it then.